libs/blocks/card/card.js

```diff
@@ -94,25 +107,32 @@ const addFooter = (links, container) => {
 };
 
 const init = (el) => {
+  const { miloLibs, codeRoot } = getConfig();
+  const base = miloLibs || codeRoot;
+  loadStyle(`${base}/deps/caas.css`);
+
   const section = el.closest('.section');
+  section.classList.add('milo-card-section');
   const row = el.querySelector(':scope > div');
```

Comment:
You could tap into the caas styles and add section.dark support here if that's a requirement. ex... 
`if (section.classList.contains('dark')) section.classList.add('consonant-u-themeDarkest');`

Reply:
Thanks for the tip but I think I'll leave it out. The XD doesn't include a dark version so we wouldn't have anything to verify it against